java/client/src/org/openqa/selenium/Cookie.java

```diff
@@ -283,8 +314,14 @@ public Builder isHttpOnly(boolean httpOnly) {
       return this;
     }
 
+    public Builder sameSite(String sameSite) {
+      this.sameSite = sameSite;
+      return this;
+    }
+
     public Cookie build() {
-      return new Cookie(name, value, domain, path, expiry, secure, httpOnly);
+      return new Cookie(
+          name, value, domain, path, expiry, secure, httpOnly, sameSite);
```

Comment:
Hi @gjyalpha,

the JSON key for new samesite cookie capability in WebDriver draft is like 'samesite' (https://w3c.github.io/webdriver/#cookies) and not as 'sameSite'. Currently am not sure as the discussion is still going about cookie normative reference in github/webdriver.

Reply:
Yes, in my commit above 2a0652c, I changed the JSON key from 'sameSite' to 'samesite'. Well, I guess I should way until the discussion is closed. My coworker is actually proposing to change it to 'sameSite' in the spec. Let's see.

Reply:
Hi @gjyalpha,

I have tested cookie parameters settings JSON key as 'samesite' and 'sameSite' in chrome 80 beta. i've observed that it accepting JSON key only as 'sameSite'. if i provide 'samesite' i am unable to see attribute in log.

If possible, can you please test from your end.

Thanks,
Harsha

Reply:
hi @gjyalpha,

The change has been done with commit https://github.com/w3c/webdriver/pull/1477.
The JSON is now as 'sameSite'.

Reply:
Yes. I'm fixing this in https://github.com/SeleniumHQ/selenium/pull/8265.